### ISSUE TITLE
Consider adding AppWorld to the list

### ISSUE THREAD

**Issue by HarshTrivedi** [hyp1231/awesome-llm-powered-agent#37]
Thanks for setting up this repository! Please consider adding AppWorld to it.

🔗 Website: https://appworld.dev/
📄 Paper: https://arxiv.org/abs/2407.18901
🐦 Tweet: https://x.com/harsh3vedi/status/1818311843976233198
💬 Blog: https://appworld.dev/blog
🎬 Video(s): https://appworld.dev/video
🌎 Code: https://github.com/stonybrooknlp/appworld
🧭 Data (task, trajectories) explorer, playground: https://appworld.dev/task-explorer
🔍 API explorer: https://appworld.dev/api-explorer
📊 Leaderboard: https://appworld.dev/leaderboard

TLDR: Introduces AppWorld Engine, a high-fidelity execution environment of 9 day-to-day apps, operable via 457 APIs, populated with digital activities of 106 people living in a simulated world, and an associated benchmark of natural, diverse, and challenging autonomous agent tasks requiring rich and interactive coding.

I think it best fits the "Benchmark", "Web Agents", and "General Reasoning & Planning & Tool Using" sections.

Thank you!

**Comment by hyp1231** [comment]
Thanks for bringing this to our attention! I've added AppWorld #38. Please feel free to raise a PR if you would like to make any modifications.

**Comment by HarshTrivedi** [comment]
Great, thank you @hyp1231 !!
